protocols/dcutr/examples/client.rs

```diff
@@ -95,7 +95,10 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     let transport = OrTransport::new(
         relay_transport,
-        block_on(DnsConfig::system(TcpConfig::new().port_reuse(true))).unwrap(),
+        block_on(DnsConfig::system(TcpTransport::new(
```

Comment:
What do you think of splitting `DnsConfig` into `DnsTransport` and `DnsConfig` like you did for `TcpConfig`?

Reply:
:+1: Let's defer.